### DISCUSSION TITLE
field suggestion

### DISCUSSION THREAD

**Discussion by elvispdosreis** [spatie/laravel-settings#133] [General]
I would like to add some columns to the table

today we only have the name we use to record for example general.site_name
I would like to have a column that describes this parameter "Site Name"

and another column to put the default value

sample
![image](https://user-images.githubusercontent.com/15267623/166919671-60968fcd-61fc-4eb8-b7f0-cfe6dccde603.png)

**Comment by rubenvanassche** [comment]
I do want to keep this package as light as possible. By default the package already has a default value which you define in the migrations. Adding a name and a preset default value could be done with an updated SettingsModel and some extra methods in the base Settings class but that's something we're not going to suppor in the package.
